src/controller/invoice.controller.js

```diff
@@ -0,0 +1,124 @@
+import invoiceModel from "../model/invoice.model.js";
+import { validationResult } from "express-validator";
+import { response } from "express";
+
+// Obtener todas las facturas
+export const getInvoices = async (req, res = response) => {
+  try {
+    const invoices = await invoiceModel.find();
+    res.status(200).json({
+      invoices,
+    });
+  } catch (error) {
+    res.status(500).json({
+      error: "Error al obtener las facturas",
+    });
+  }
+};
+
+// Obtener una factura por su ID
+export const getInvoiceById = async (req, res = response) => {
+  const { id } = req.params;
+
+  try {
+    const invoice = await invoiceModel.findById(id);
+    if (!invoice) {
+      return res.status(404).json({
+        error: "Factura no encontrada",
+      });
+    }
+    res.status(200).json({
+      invoice,
+    });
+  } catch (error) {
+    res.status(500).json({
+      error: "Error al obtener la factura",
+    });
+  }
+};
+
+// Crear una nueva factura
+export const createInvoice = async (req, res = response) => {
+  const errors = validationResult(req);
+  if (!errors.isEmpty()) {
+    return res.status(400).json({
+      errors: errors.array(),
+    });
+  }
+
+  const { id, price, booking_id, user_id } = req.body;
+
+  try {
+    const newInvoice = new invoiceModel({
+      id,
+      price,
+      tp_status: "ACTIVE",
+      booking_id,
+      user_id,
+    });
+    await newInvoice.save();
+    res.status(201).json({
+      message: "Factura creada correctamente",
+      invoice: newInvoice,
+    });
```

Comment:
El ID no deberia ser pasado por el body, el id es un identificdor unico que genera mongo. eso esta mal.
```suggestion
  const { price, booking_id, user_id } = req.body;

  try {
    const newInvoice = new invoiceModel({
      price,
      // tp_status: "ACTIVE", esto se pone por defecto en la DB. ver el modelo de la entidad
      booking_id,
      user_id,
    });
    await newInvoice.save();
    res.status(201).json({
      message: "Factura creada correctamente", // INGLES
      invoice: newInvoice,
    });
```

porfavor recuerda trabajar en ingles.